main/e2e-testing.md

```diff
@@ -143,3 +143,142 @@ Some examples projects utilizing `@agoric/synpress` for e2e tests:
 
 - [PSM dApp E2E Tests](https://github.com/Agoric/dapp-psm/tree/main/tests/e2e)
 - [Wallet dApp E2E Tests](https://github.com/frazarshad/wallet-app/tree/main/test/e2e)
+
+## @agoric/synpress Commands
+
+The `@agoric/synpress` commands are custom Cypress commands designed for use with Keplr. They provide ready-made functions to interact with Keplr-specific elements and workflows.
+
+## `setupWallet`
```

Comment:
use `###` here and in the following subheadings unless you are using another format.